.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    root: true,
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module'
+    },
+    extends: 'eslint:recommended',
```

Comment:
I haven't used it recently, and can't remember the last time I tried. I do like the idea that we can fix small linter errors automatically. That's just one of the benefits of moving to ESLint.